src/routes/_pages/settings/instances/index.html

```diff
@@ -5,11 +5,16 @@ <h1>Instances</h1>
   <p>Instances you've logged in to:</p>
   <SettingsList label="Instances">
     {#each $loggedInInstancesAsList as instance}
-      <SettingsListItem offsetForIcon={instance.name !== $currentInstance}
-                        icon={instance.name === $currentInstance ? '#fa-star' : ''}
-                        href="/settings/instances/{instance.name}"
-                        label={instance.name}
-                        ariaLabel="{instance.name} {instance.name === $currentInstance ? '(current instance)' : ''}" />
+    <SettingsListRow>
+      <SettingsListButton href="/settings/instances/{instance.name}"
+                          label={instance.name}
+                          ariaLabel="{instance.name} {instance.name === $currentInstance ? '(current instance)' : ''}" />
+      <button class="instance-switcher-button" on:click="onSwitchToThisInstance(event, instance.name)">
+        <svg class="instance-switcher-button-svg">
+          <use xlink:href="{instance.name === $currentInstance ? '#fa-star' : '#fa-star-o'}" />
+        </svg>
+      </button>
```

Comment:
This should have a "pressed" state and an aria label to indicate what it does. I also like using titles so people can hover and it says what the button does. (For instance, "switch to this instance" and "switch away from this instance".)